android/src/main/java/com/swmansion/rnscreens/InsetsObserverProxy.kt

```diff
@@ -57,10 +57,8 @@ object InsetsObserverProxy : OnApplyWindowInsetsListener {
         }
     }
 
-    fun unregister() {
-        eventSourceView.get()?.takeIf { hasBeenRegistered }?.let {
-            ViewCompat.setOnApplyWindowInsetsListener(it, null)
-        }
+    fun unregisterOnView(view: View) {
+        ViewCompat.setOnApplyWindowInsetsListener(view, null)
```

Comment:
Note for future me:

This change broke formSheet behaviour with keyboard.
By calling it this way, we remove inset listener from `view` even if `InsetObserverProxy` has not been registered, that means if someone else registered on `view` we overwrite the listener to null.